backend/docs/adr/20240129-use-authentication.md

```diff
@@ -0,0 +1,53 @@
+# Authentication management
+
+- **Status**: proposed
+- **Deciders**: Johann Meza
+- **Date**: 01/29/2024
+
+## Context and Problem Statement
+
+In the development of a web platform, we need a robust and efficient authentication system to guarantee the security and proper management of user identity.
+
+Currently, we face challenges with the traditional authentication approach, which is complex to manage, presents security risks, and does not provide an optimal experience for users.
+
+With Auth0, we can centralize user management, improve security, and offer a smoother login experience for our users.
```

Comment:
Esto en realidad va en los pros de Auth0. En el Context and Problem Statement se expone el problema o el contexto, sin hablar de las formas de implementar la solución.